GOVERNANCE.md

```diff
@@ -28,75 +27,74 @@ Flux strives to operate independently of specific partisan interests, and for de
 
 ### Code of Conduct
 
-The Flux community adheres to the CNCF Code of Conduct <https://github.com/cncf/foundation/blob/master/code-of-conduct.md>.
+[Code of Conduct](CODE_OF_CONDUCT.md) violations by community members will be discussed and resolved on the [private Maintainer mailing list](mailto:cncf-flux-maintainers@lists.cncf.io). If the reported CoC violator is a Maintainer, the Maintainers will instead designate two Maintainers to work with CNCF staff in resolving the report.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a _Flux_ [Oversight Committee] member.
+### Meetings
 
-If no conclusion can be reached in meditation, such issues can be escalated to the CNCF mediator, Mishi Choudhary <mishi@linux.com>, in which case CNCF may choose to intervene.
+Time zones permitting, Maintainers are expected to participate in the [public
+developer meeting](README.md#meetings).
 
-## Roles in the Flux Community
+Maintainers will also have closed meetings in order to discuss security reports
+or Code of Conduct violations.  Such meetings should be scheduled by any
+Maintainer on receipt of a security issue or CoC report.  All current Maintainers
+must be invited to such closed meetings, except for any Maintainer who is
+accused of a CoC violation.
 
-See [community-roles.md]
+Note: Refer to [our security process](https://fluxcd.io/security) for more information on how security issues are handled in particular.
+
+## Roles and Process in the Flux Community
+
+See
+
+- [community-roles.md] for the definition of roles in our community and
+- [PROCESS.md] to understand how to interact with Flux Community processes
```

Comment:
Looks like this is missing a link at the bottom of the page, due to which, in the markdown preview PROCESS.md is not a hyperlink.

Reply:
:+1: